webapp/channels/src/sass/layout/_module.scss

```diff
@@ -10,4 +10,3 @@
 @import 'team-button';
 @import 'team-sidebar';
 @import 'webhooks';
-@import 'channel-icon';
```

Comment:
Did you intend to re-add this? I see you re-added the file at some point last week but you removed this line at the same time

Reply:
Yup, adding it back.